Library/Homebrew/cleanup.rb

```diff
@@ -36,10 +42,18 @@ def self.cleanup_logs
       end
     end
 
-    def self.cleanup_cellar
-      Formula.installed.each do |formula|
-        cleanup_formula formula
+    def self.cleanup_cellar(formulae = Formula.installed)
```

Comment:
The use case of this is to have the command not duplicate the continue on fail logic. See https://github.com/Homebrew/brew/pull/2374/files#diff-8cc42f61c29a8f960250394cbc3319c4L24